modules/light-clients/08-wasm/types/msgs.go

```diff
@@ -33,3 +35,70 @@ func (m MsgStoreCode) GetSigners() []sdk.AccAddress {
 	}
 	return []sdk.AccAddress{signer}
 }
+
+type QueryMsg struct {
```

Comment:
Wondering if these might be best suited in `vm`?. `msgs.go` was inspired by wasmvm's placement yeah? Though their `types/msgs.go` doesn't really have the typical interpretation as found in modules (implementing sdk.Msg and all).

Reply:
Agree on highlighting this! Would be nice to find a nice home for these specific messages. I know `msgs.go` is usually used for implementations of `sdk.Msg` (anything that gets routed through a grpc service). 

But these messages are kind of special. I don't have a problem keeping them in `msgs.go` personally, but I could also see value in some other dedicated file like `api.go` or similar which signals their intent. Also having them separate is probably a good call for maintainabiilty/visibilty, as these msgs cannot really change once the API is published

Reply:
I moved them to a file called `contract_api.go`. I hope the filename is fine for you!